concrete/single_pages/dashboard/files/sets.php

```diff
@@ -1,178 +1,241 @@
-<?php defined('C5_EXECUTE') or die("Access Denied.");
+<?php
 
-$ih = Core::make('helper/concrete/ui');
-$dh = Core::make('helper/date');
+defined('C5_EXECUTE') or die("Access Denied.");
+
+use Concrete\Core\File\FileList;
+use Concrete\Core\File\Set\Set;
+use Concrete\Core\Localization\Service\Date;
+use Concrete\Core\Support\Facade\Application;
+use Concrete\Core\Support\Facade\Url;
+use Concrete\Core\Validation\CSRF\Token;
+
+/** @var int $fsType */
+/** @var Set $fs */
+/** @var Token $validation_token */
+$app = Application::getFacadeApplication();
+$app->make('helper/concrete/ui');
+/** @var Date $dh */
+$dh = $app->make(Date::class);
 
 ?>
-<?php if ($this->controller->getTask() == 'view_detail') {
-    ?>
+
+<?php if ($this->controller->getTask() == 'view_detail'): ?>
 
     <script type="text/javascript">
-        deleteFileSet = function () {
-            if (confirm('<?=t('Are you sure you want to permanently remove this file set?')?>')) {
-                location.href = "<?=$view->url('/dashboard/files/sets', 'delete', $fs->getFileSetID(),
```

Comment:
Note: changing <?= to <?php echo isn't really doing anyone any favors here. I'm not going to raise a fuss over it but just something to be aware of.